server/src/main/java/keywhiz/service/resources/automation/v2/LogArgumentsModule.java

```diff
@@ -0,0 +1,11 @@
+package keywhiz.service.resources.automation.v2;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.matcher.Matchers;
+
+public class LogArgumentsModule extends AbstractModule {
+  @Override
+  protected void configure() {
+    bindInterceptor(Matchers.any(), Matchers.annotatedWith(LogArguments.class), new LogArgumentsImpl());
+  }
+}
```

Comment:
Seeing code like this that's so powerful yet so concise makes me happy. Yay Guice!